includes/class-amp-story-post-type.php

```diff
@@ -323,6 +347,55 @@ public static function admin_enqueue_scripts( $hook ) {
 
 	}
 
+	/**
+	 * Load font from story data.
+	 *
+	 * @param WP_Post $post Post Object.
+	 */
+	public static function load_fonts( $post ) {
+		$post_story_data = json_decode( $post->post_content_filtered, true );
+		$g_fonts         = [];
+		if ( $post_story_data ) {
+			foreach ( $post_story_data as $page ) {
+				foreach ( $page['elements'] as $element ) {
+					$font = AMP_Fonts::get_font( $element['fontFamily'] );
+
+					if ( $font && isset( $font['gfont'] ) && $font['gfont'] ) {
+						if ( isset( $g_fonts[ $font['name'] ] ) && ! in_array( $element['fontWeight'], $g_fonts[ $font['name'] ], true ) ) {
+							$g_fonts[ $font['name'] ][] = $element['fontWeight'];
+						} else {
+							$g_fonts[ $font['name'] ][] = $element['fontWeight'];
```

Comment:
The `if` and `else` contents seem to be identical (both `$g_fonts[ $font['name'] ][] = $element['fontWeight'];`).

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3998/commits/8bbb28b930ae5f0a2b85bf7f8f1acb7d5c13100d